impeller/display_list/display_list_dispatcher.cc

```diff
@@ -381,11 +389,12 @@ static std::optional<Rect> ToRect(const SkRect* rect) {
 void DisplayListDispatcher::saveLayer(const SkRect* bounds,
                                       const flutter::SaveLayerOptions options,
                                       const flutter::DlImageFilter* backdrop) {
+  auto paint = options.renders_with_attributes() ? paint_ : Paint{};
   if (backdrop) {
-    UNIMPLEMENTED;
+    canvas_.SaveLayer(paint, ToRect(bounds), ToImageFilterProc(backdrop));
+  } else {
+    canvas_.SaveLayer(paint, ToRect(bounds));
```

Comment:
`canvas_.SaveLayer` takes an optional backdrop. Does having a special call omitting the argument save much?

Reply:
Done.